beacon-chain/casper/validator.go

```diff
@@ -277,3 +326,22 @@ func minEmptyValidator(validators []*pb.ValidatorRecord) int {
 	}
 	return -1
 }
+
+// DeepCopyValidators creates a fresh new validator set by copying all the validator details
+// from the old validator set.
```

Comment:
Can you add a comment as to where this could be useful? e.g. as mentioned above we could use this func in the state recalcs

Reply:
This isn't a deep copy, since we're reusing the values for each property. For example, modifying one byte in `RandaoCommitment` would modify the original one as well. For this purpose, I think that's fine, but we the name should just be `CopyValidators`.